### PR TITLE
Qt: fix never played nonsense

### PR DESCRIPTION
**Pull request by Megamouse**

### CODE REVIEW

File: rpcs3/rpcs3qt/localized.cpp [hunk 1]
```diff
@@ -4,29 +4,24 @@ Localized::Localized()
...
+QString Localized::GetVerboseTimeByMs(quint64 elapsed_ms, bool show_days) const
 {
-	if (elapsed_ms <= 0)
-	{
-		return tr("Never played");
-	}
-
-	const qint64 elapsed_seconds = (elapsed_ms / 1000) + ((elapsed_ms % 1000) > 0 ? 1 : 0);
-	qint64 hours = elapsed_seconds / 3600;
+	const quint64 elapsed_seconds = (elapsed_ms / 1000) + ((elapsed_ms % 1000) > 0 ? 1 : 0);
```
Comment (bevanweiss) [inline rpcs3/rpcs3qt/localized.cpp:9]: Should this have been '> 499' to round to the nearest second?

File: rpcs3/rpcs3qt/update_manager.cpp [hunk 2]
```diff
@@ -180,7 +180,7 @@ bool update_manager::handle_json(bool automatic, bool check_only, const QByteArr
 			.arg(cur_str)
 			.arg(latest["version"].toString())
 			.arg(lts_str)
-			.arg(localized.GetVerboseTimeByMs(diff_msec, true));
+			.arg(localized.GetVerboseTimeByMs(std::abs(diff_msec), true));
```
Comment (bevanweiss) [inline rpcs3/rpcs3qt/update_manager.cpp:183]: std::max(0,diff_msec) seems more appropriate here

File: rpcs3/rpcs3qt/localized.cpp (hunk 1 above)
Comment (Megamouse) [reply rpcs3/rpcs3qt/localized.cpp:9]: No

File: rpcs3/rpcs3qt/update_manager.cpp (hunk 2 above)
Comment (Megamouse) [reply rpcs3/rpcs3qt/update_manager.cpp:183]: No

**Pull request merged**